jetty-core/jetty-http3/jetty-http3-tests/src/test/java/org/eclipse/jetty/http3/tests/AbstractClientServerTest.java

```diff
@@ -63,34 +63,41 @@ public class AbstractClientServerTest
 
     protected void start(Handler handler) throws Exception
     {
-        ServerQuicConfiguration quicConfiguration = newServerQuicConfiguration();
+        ServerQuicConfiguration quicConfiguration = newServerQuicConfiguration(false);
         prepareServer(quicConfiguration, new HTTP3ServerConnectionFactory(quicConfiguration));
         server.setHandler(handler);
         server.start();
         startClient();
     }
 
-    private ServerQuicConfiguration newServerQuicConfiguration()
+    protected void start(Session.Server.Listener listener) throws Exception
     {
-        SslContextFactory.Server sslServer = new SslContextFactory.Server();
-        sslServer.setKeyStorePath("src/test/resources/keystore.p12");
-        sslServer.setKeyStorePassword("storepwd");
-        return new ServerQuicConfiguration(sslServer, workDir.getEmptyPathDir());
+        startServer(false, listener);
+        startClient();
     }
 
-    protected void start(Session.Server.Listener listener) throws Exception
+    protected void start(boolean needClientAuth, Session.Server.Listener listener) throws Exception
```

Comment:
The `boolean` is quite arbitrary, and does not help much when looking at tests that call `start(true, ...)`.

Can you please revert this, and add for the tests that need it, this line, just after calling `start(...)`:

`connector.getQuicConfiguration().getSslContextFactory().setNeedClientAuth(true);`

Reply:
Good suggestion, let me do this.